docs/vectorizer-quick-start.md

```diff
@@ -0,0 +1,130 @@
+    ```
+
+4. **Create a vectorizer for `blog`**
+
+    ```sql
+    SELECT ai.create_vectorizer(
+       'blog'::regclass,
+       destination => 'blog_contents_embeddings',
+       embedding => ai.embedding_openai('text-embedding-3-small', 768),
+       chunking => ai.chunking_recursive_character_text_splitter('contents'),
+       scheduling => ai.scheduling_none()
+    );
+    ```
+
+5. **Run the vectorizer worker**
+
+   When you install pgai on Timescale Cloud, vectorizers are run automatically using TimescaleDB scheduling. 
+   For self-hosted, you run a pgai vectorizer worker so the vectorizer can process the data in `blog`. 
+   
+   In a new terminal, start the vectorizer worker:
+   ```shell
+   docker-compose up -d vectorizer-worker
+   ```
+   You see the vectorizer running:
+   ```shell
+   [+] Running 11/11
+   ✔ vectorizer-worker Pulled                                                                                                                                           22.3s 
+     ✔ a480a496ba95 Pull complete                                                                                                                                        6.0s 
+     ✔ f281ad68d612 Pull complete                                                                                                                                        6.1s 
+     ✔ aa77a90b11d9 Pull complete                                                                                                                                        6.5s 
+     ✔ 0dd753b657c9 Pull complete                                                                                                                                        6.5s 
+     ✔ 38e39325ccaf Pull complete                                                                                                                                        6.5s 
+     ✔ 8d50dded1d76 Pull complete                                                                                                                                        6.5s 
+     ✔ 5a73fecc1c3f Pull complete                                                                                                                                        6.5s 
+     ✔ 03f1e7e22d43 Pull complete                                                                                                                                       20.0s 
+     ✔ 33c61f413670 Pull complete                                                                                                                                       20.0s 
+     ✔ dc491244f87c Pull complete                                                                                                                                       20.0s 
+     [+] Running 1/1
+     [+] Running 1/2ai-vectorizer-worker-1  
+    ```
```

Comment:
Is this shell output relevant? We don't have it for starting the db container, I think it just bloats the tutorial a bit.
As a user I might skim through the steps of such a quick start and if it looks to complicated I just wont do it. I think having a lot of console output contributes to that.

Reply:
Removed